include/ofi_util.h

```diff
@@ -760,6 +763,50 @@ static inline void ofi_cntr_inc(struct util_cntr *cntr)
 	cntr->cntr_fid.ops->add(&cntr->cntr_fid, 1);
 }
 
+static inline void ofi_ep_peer_tx_cntr_inc(struct util_ep *ep, uint8_t op)
+{
+	struct util_cntr *cntr;
+	int cntr_index;
+
+	cntr_index = ofi_get_cntr_index_from_tx_op(op);
+	cntr = ep->cntrs[cntr_index];
+	if(cntr)
+		cntr->peer_cntr->owner_ops->inc(cntr->peer_cntr);
+}
+
+static inline void ofi_ep_peer_tx_cntr_incerr(struct util_ep *ep, uint8_t op)
+{
+	struct util_cntr *cntr;
+	int cntr_index;
+
+	cntr_index = ofi_get_cntr_index_from_tx_op(op);
+	cntr = ep->cntrs[cntr_index];
+	if(cntr)
```

Comment:
space after if - applies to all 4 peer functions

Reply:
Fixed.